src/dbt_mcp/config/config.py

```diff
@@ -89,11 +89,23 @@ def actual_prod_environment_id(self) -> int | None:
 
     @field_validator("disable_tools", mode="before")
     @classmethod
-    def parse_disable_tools(cls, env_var: str | list) -> list[ToolName]:
+    def parse_disable_tools(cls, env_var: str | None) -> list[ToolName]:
         if not env_var:
             return []
-        assert isinstance(env_var, str)
-        return [ToolName(tool.strip()) for tool in env_var.split(",") if tool.strip()]
+        errors: list[str] = []
+        tool_names: list[ToolName] = []
+        for tool_name in env_var.split(","):
+            tool_name_stripped = tool_name.strip()
+            try:
```

Comment:
We could skip empty strings, no?

```suggestion
            tool_name_stripped = tool_name.strip()
            if tool_name_stripped == "":
                continue
            try:
```